### PR TITLE
Remove mutate image ref

### PR DESCRIPTION
**Pull request by jkhelil**
# Changes
Remove mutate_image_container_image reference

Fixes #<insert issue number here>
https://github.com/shipwright-io/operator/issues/127
-->

# Submitter Checklist

- [ ] Includes tests if functionality changed/was added
- [ ] Includes docs if changes are user-facing
- [ ] [Set a kind label on this PR](https://prow.k8s.io/command-help#kind)  
- [ ] Release notes block has been filled in, or marked NONE

See [the contributor guide](https://github.com/shipwright-io/build/blob/master/CONTRIBUTING.md)
for details on coding conventions, github and prow interactions, and the code review process.

# Release Notes
```release-note
NONE
```

### CODE REVIEW

**Comment by jkhelil** [comment]
docker pull ghcr.io/shipwright-io/build/image-processing
Using default tag: latest
Error response from daemon: Head https://ghcr.io/v2/shipwright-io/build/image-processing/manifests/latest: unauthorized


lets wait the package going public before merging this

File: kodata/release.yaml
Comment (SaschaSchwarze0) [inline kodata/release.yaml:151]: Such a tag does not exist.

**Review by SaschaSchwarze0 — CHANGES_REQUESTED** [review]
I think we will need to keep this PR open until v0.12 release is happening. Because only then we have the image-processing image in a release.

Anyway, good that we have it already prepared.

**Comment by qu1queee** [comment]
From the Shipwright refinement meeting, PR is on-hold until the new release v0.12.0 is ready.

**Comment by adambkaplan** [comment]
/hold

Per @qu1queee's comment.

Since the v0.12 release will lead to some refactoring of the operator, should we cut a "release candidate" of the build before we add it here?

**Comment by adambkaplan** [comment]
We also have nightly releases yamls - those could be used ahead of a release as well.

**Pull request closed without merging**
